tests/internal/scripts/ccuda/cuda_grid.py

```diff
@@ -0,0 +1,30 @@
+from pyccel.decorators import kernel, types
+from pyccel import cuda
+
+@kernel
+@types('int[:]')
+def func_1d(a):
```

Comment:
it would be better to call all of these `kernel` functions

Reply:
Done.